### ISSUE TITLE
[a11y and e2e] Make the tree a11y and e2e friendly

### ISSUE THREAD

**Issue by unlikelyzero** [nasa/openmct#5587]
**Is your feature request related to a problem? Please describe.**
At present, our tree is "unfriendly" to our e2e tests and not accessible by any definition.

**Describe the solution you'd like**
We should implement the tree role to our tree using https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tree_role

Item work:
- [x] Research aria tree role
- [x] Implement aria tree role
- [ ] Validate that aria tree role matches some WCAG standard (somehow)
- [x] Update e2e tests
- [ ] Validate that the playwright codegen works with the tree elements
- [ ] Update e2e test best practices with how to use the tree

**Comment by unlikelyzero** [comment]
Implemented here https://github.com/nasa/openmct/pull/5643
